build.gradle

```diff
@@ -322,5 +331,48 @@ task buildMap(type: JavaExec, dependsOn: 'build') {
     classpath = project.sourceSets.main.runtimeClasspath
 }
 
+task codeCoverageReport(type: JacocoReport) {
+    executionData fileTree(project.rootDir.absolutePath).include("**/build/jacoco/*.exec")
+    sourceSets sourceSets.main
+    subprojects.each {
+        sourceSets it.sourceSets.main
+    }
+    // excludes = [*]
+    // includes = ['src/team3player/**']
+    afterEvaluate {
+        classDirectories.setFrom(files(classDirectories.files.collect {
+            fileTree(dir: it, include: '**/team3player/**')
+        }))
+    }
+    reports {
+        html.enabled true
+    }
+}
+
+codeCoverageReport.dependsOn {
+    subprojects*.test
+}
+
+task testCodeCoverage(type: JacocoCoverageVerification) {
```

Comment:
This adds the gradle command testCodeCoverage which will exit with a status code indicating a pass if the overall coverage is above 10% or return a failure if it is under %10.